### DISCUSSION TITLE
Move specific stylemanager category to another panel or outside stylemanager

### DISCUSSION THREAD

**Discussion by jloguercio** [GrapesJS/grapesjs#4064] [Q&A]
How can i move an specific stylemanager category for example "general" or "typography" to another panel ?, there is a way that can filter or something like blocks categories?

Thanks!

**Comment by artf** [comment]
I think you might want to try to simply move the category DOM element via JS once it's rendered but I have no guarantee about the side effects that could generate.
In the alternative, from the [latest version](https://github.com/artf/grapesjs/releases/tag/v0.18.1), you can create your [own Style Manager UI](https://grapesjs.com/docs/modules/Style-manager.html#customization) via API.
